docs/tools/brilift.md

```diff
@@ -0,0 +1,63 @@
+Cranelift Compiler
+==================
+
+Brilift is a compiler from Bril to native code using the [Cranelift][] code generator.
+It supports [core Bril][core] only.
+
+Brilift is an ahead-of-time compiler.
+It emits `.o` files and also provides a simple run-time library.
+By linking these together, you get a complete native executable.
+
+[cranelift]: https://github.com/bytecodealliance/wasmtime/tree/main/cranelift
+[core]: ../lang/core.md
+
+Build
+-----
+
+Brilift is a Rust project using the [bril-rs][] library.
+You can build it using [Cargo][]:
+
+
+    $ cd brilift
+    $ cargo build
+    $ cargo install  # If you want the executable on your $PATH.
```

Comment:
This is deprecated and wont work. Use `cargo install --path .` instead. Or `cargo install --path brilift` without the `cd brilift`. Or have something like `make install`. I don't think it's worth asking a potential user to run `cargo build`. If they want to run this code with out installing it, they would do this via `run` instead, otherwise if they want the executable, `install` is going to rebuild the project anyways in release mode.

Reply:
Ah, thanks! Didn't realize this feature was removed. Updated!